plugins/cybereason/icon_cybereason/util/api.py

```diff
@@ -82,10 +90,14 @@ def get_sensor_details(self, identifier: str) -> dict:
                 cause=f"Multiple sensors found using identifier: {identifier}.",
                 assistance="Please provide a unique identifier and try again.",
             )
-
         return sensors.get("sensors")[0]
 
-    def get_malop(self, malop_guid: str) -> dict:
+    def archive_sensor(self, sensor_ids: list, argument: str) -> Dict[str, Any]:
```

Comment:
can you use `List` imported from typing like in other methods,
and can you specify what's in the list, you can use `List[Any]` if there is everything, or `List[Dict]` if list contains a dictionary, etc.